server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java

```diff
@@ -199,7 +213,7 @@ public Object executeAggregation(Signature maybeUnboundSignature,
         );
         for (var argType : actualArgumentTypes) {
             if (argType.storageSupport() == null) {
-                return aggregationFunction.terminatePartial(
+                return terminatePartialAggFunction.terminatePartial(
```

Comment:
Wouldn't it make sense to keep some `terminatePartial` coverage for the `aggregationFunction` (iter->final) in addition to the second variant? (partial->final)

Reply:
Do you think we should it with randomness, or execute both and compare the results?

Reply:
I'd tend to randomness but either way is fine

Reply:
I came up with this: https://github.com/crate/crate/pull/16532 which seems simple enough, and since it's unit tests, I thought it won't hurt to execute both paths and assert they yield the same result.